source/guides/core.rst

```diff
@@ -14,8 +14,15 @@ Development Process
    /process/release-process*
    /process/accepting-pull-request*
    /process/community-systems*
-   /process/documentation-guidelines*
+   /process/documentation-guidelines.md
 
+Documentation Style Guide
+=========================
+
+.. toctree::
+   :maxdepth: 3
+
+   /process/sg_mattermost-doc-style.rst
```

Comment:
Thanks @JeffSchering makes sense, we'll leave file naming out of the guide for now, and we can think about it later--along with a strategy for redirecting out-of-date pages.